packages/module/src/catalog/__tests__/QuickStartTileDescription.spec.tsx

```diff
@@ -1,8 +1,8 @@
-import * as React from 'react';
 import { Popover } from '@patternfly/react-core';
 import { shallow } from 'enzyme';
 import { getQuickStarts } from '../../data/test-utils';
 import QuickStartTileDescription from '../QuickStartTileDescription';
+import React from 'react';
```

Comment:
Just wanted to confirm since it was moved from line 1, but I assume we need to keep the react import here?

Reply:
Ah I think we can actually remove these for now for consistency. These two files were the ones that I ran into issues with enzyme compatibility, so had added back in the import to try to fix them, but ended up ignoring the suites.